### DISCUSSION TITLE
Copilot not working and missing a token even after relogging and reinstalling

### DISCUSSION THREAD

**Discussion by ohusq** [community/community#51663] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

`[INFO] [auth] [2023-04-01T08:04:13.295Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-01T08:04:13.303Z] GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"`

I've used Copilot around a week ago and it worked perfect fine. Did they ban it in some countries?

**Comment by Akash1134** [comment, marked as answer]
Hi @66-75-63-6B-79-6F-75 ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can continue to use other GitHub services including your Pro plan. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear.

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them. Thanks!

**Reply by ohusq** [reply]
Thank you for the help, but I'm from the Netherlands and not from Iran. I'll open a ticket and close this thread after having my issue resolved
